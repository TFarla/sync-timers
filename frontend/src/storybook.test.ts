import initStoryshots from '@storybook/addon-storyshots';

// eslint-disable-next-line @typescript-eslint/no-unsafe-call
initStoryshots();