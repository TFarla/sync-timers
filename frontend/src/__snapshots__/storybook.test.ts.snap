// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Timers Test 1`] = `<ul />`;
